src/views/List.jsx

```diff
@@ -16,37 +20,61 @@ export function List({ data, isLoading, listToken }) {
 	if (isLoading) {
 		return <BeatLoader color="black" loading={isLoading} size={15} />;
 	}
+
+	const handleModal = () => {
```

Comment:
```suggestion
	const openModal = () => {
```

nitpick 😉 : Just a suggestion

Reply:
Updated